python/pyspark/sql/tests.py

```diff
@@ -4124,7 +4126,7 @@ def test_vectorized_udf_timestamps(self):
         data = [(0, datetime(1969, 1, 1, 1, 1, 1)),
                 (1, datetime(2012, 2, 2, 2, 2, 2)),
                 (2, None),
-                (3, datetime(2100, 3, 3, 3, 3, 3))]
+                (3, datetime(2100, 4, 4, 4, 4, 4))]
```

Comment:
Just wondering if changing these values made a difference somewhere?

Reply:
I'll revert it.